manim_slides/convert.py

```diff
@@ -707,6 +730,11 @@ def callback(ctx: Context, param: Parameter, value: bool) -> None:
     help="Use the template given by FILE instead of default one. "
     "To echo the default template, use '--show-template'.",
 )
+@click.option(
+    "--one_file",
```

Comment:
```suggestion
    "--one-file",
```

Just for the convention :-)

Reply:
I would actually copy the same logic used for the `--offline` option, see suggestion below.